contracts/src/v0.8/l2ep/dev/scroll/ScrollSequencerUptimeFeed.sol

```diff
@@ -142,24 +136,23 @@ contract ScrollSequencerUptimeFeed is
   /// @param status Sequencer status
   /// @param timestamp Block timestamp of status update
   function updateStatus(bool status, uint64 timestamp) external override {
-    FeedState memory feedState = s_feedState;
```

Comment:
Shit should probably also stay, done to reduce gas usage. Stuff like this is why I love having a Foundry test suite with a gas snapshot, you will see the impact of any change you make on the gas on an individual function level. That would have shown the gas diff here clearly, which I expect is an increase in cost for this function.